packages/agoric-cli/lib/chain-config.js

```diff
@@ -108,6 +108,7 @@ export function finishCosmosGenesis({ genesisJson, exportedGenesisJson }) {
   // TODO: This needs much more support to preserve contract state
   // between exports in order to be able to carry forward IBC conns.
   genesis.app_state.ibc = initState.ibc;
+  genesis.app_state.capability = initState.capability;
```

Comment:
I'm curious what this is bout.

But I don't know enough to think it's wrong.

Reply:
We need to reset the cap state (since it is tied directly to the ibc state), otherwise it is inconsistent and halts the chain.